lib/ansible/module_utils/facts/system/aix.py

```diff
@@ -0,0 +1,257 @@
+# AIX specific facts
+# Copyright: (c) 2018, Christian Tremel (@flynn1973)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+import sys
+import shlex
+import os
+import platform
+import re
+import itertools
+import subprocess
+try:
+    import json
+except ImportError:
+    import simplejson as json
+
+
+from ansible.module_utils.facts.utils import get_file_content
+
+
+from ansible.module_utils.facts.collector import BaseFactCollector
+
+
+class AIXSystemFactCollector(BaseFactCollector):
+    """
+    Some handy AIX facts, more or less nice to haves
+    """
+    name = 'aix_system_facts'
+    _fact_ids = set()
+    _platform = 'AIX'
+
+    def get_lssrc_facts(self, module=None, collected_facts=None):
+        """
+        'lssrc': [
+                  {
+                   'Subsystem': 'prngd',
+                   'PID': '3211316',
+                   'Group': 'prng',
+                   'Status': 'active'
+                   },
+                  {
+                   'Subsystem': 'syslogd',
+                   'PID': '4849686',
+                   'Group': 'ras',
+                   'Status': 'active'
+                   }
+                   ],
+        """
+        lssrc_facts = {}
+
+        lssrc_facts['lssrc'] = []
+
+        lssrc = []
+
+        rc, out, err = self.module.run_command(["/usr/bin/lssrc", "-a"])
+        if rc != 0:
+            self.module.fail_json(
+                msg="ERROR: Could not run lssrc command",
+                rc=rc,
+                err=err)
+        firstline = True
+        for line in out.splitlines():
+            for line in out.splitlines():
```

Comment:
Isn't duplicate this loop in here?
I also think we could simplify it here starting after the first line
```python
for line in out.splitlines()[1:]
```